app/services/item_service.rb

```diff
@@ -0,0 +1,10 @@
+class ItemService
+  def self.all_items
+    response = conn.get("/api/v1/items")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.conn
+    Faraday.new("http://localhost:3000")
+  end
```

Comment:
This method is exactly the same as `self.conn` in `MerchantService`. Could these independent services inherit from a more generic service (ie `RailsEngineService`), where we could move this method to DRY things up?